api/src/user/mutations/authenticate.js

```diff
@@ -147,10 +148,15 @@ export const authenticate = new mutationWithClientMutationId({
         throw new Error(i18n._(t`Unable to authenticate. Please try again.`))
       }
 
-      const token = tokenize({ parameters: { userKey: user._key } })
+      const token = tokenize({
+        expiresIn: '15m',
+        parameters: { userKey: user._key },
+        secret: String(AUTHENTICATED_KEY),
+      })
+
       const refreshToken = tokenize({
+        expiresIn: '7d',
```

Comment:
'15m' and '7d' appear throughout multiple files. Should be declared in one place (e.g. env secrets)